playgrounds/basic/package.json

```diff
@@ -9,7 +9,15 @@
 	},
 	"devDependencies": {
 		"@sveltejs/adapter-auto": "workspace:*",
+		"@sveltejs/adapter-cloudflare": "workspace:*",
+		"@sveltejs/adapter-cloudflare-workers": "workspace:*",
+		"@sveltejs/adapter-netlify": "workspace:*",
+		"@sveltejs/adapter-node": "workspace:*",
+		"@sveltejs/adapter-static": "workspace:*",
+		"@sveltejs/adapter-vercel": "workspace:*",
+		"@sveltejs/amp": "workspace:*",
 		"@sveltejs/kit": "workspace:*",
+		"@sveltejs/package": "workspace:*",
```

Comment:
would this use package? if so, I think we'd need to add a script for it in this file as well?

Reply:
oh, I hadn't scrolled down enough to see you're also adding a playground for package :laughing: then yeah, we probably don't need this here...

Reply:
Fixed!